db/makeMongoData.js

```diff
@@ -0,0 +1,65 @@
+const fs = require('fs');
+const faker = require('faker');
+
+const pickBackers = (numBackers) => {
+  const backers = {};
+  let count = 0;
+  while (count < numBackers) {
+    const backerId = Math.floor(Math.random() * 1000);
+    if (!backers[backerId]) {
+      backers[backerId] = true;
+      count++;
+    }
+  }
+  return Object.keys(backers).map(el => +el);
+};
+
+const writeProjects = () => {
+  let notFull = true;
+  while (j < 1e7 && notFull) {
+    notFull = projects.write(JSON.stringify({
```

Comment:
What does 1e7 do here?

Reply:
1e7 is just shorthand notation for 1 * 10^7 (10 million).